packages/amazonq/test/unit/codewhisperer/service/securityScanHandler.test.ts

```diff
@@ -320,4 +321,37 @@ describe('securityScanHandler', function () {
             await assert.rejects(() => pollPromise, SecurityScanTimedOutError)
         })
     })
+
+    describe('generateScanName', function () {
+        const clientId = 'ffffffff-ffff-ffff-ffff-ffffffffffff'
+
+        it('should generate scan name for FILE_AUTO scope', function () {
```

Comment:
you could probably drop the should on all of these tests so they read like:
```suggestion
        it('generates scan name for FILE_AUTO scope', function () {
```